src/raftstore/store/peer_storage.rs

```diff
@@ -700,6 +706,8 @@ impl PeerStorage {
         true
     }
 
+    /// Get a snapshot. Returns `SnapshotTemporarilyUnavailable` if there is no unavailable
```

Comment:
@hicqu When used for description, comments should be descriptive rather than imperative. It means you should use the third person instead of the second person writing the comments. 
- Opens the file √
- Open the file ×
Even though you are not sure whether to use the second or the third person, you should be consistent in this issue for this document.